### ISSUE TITLE
Update README.md about the project being dead

### ISSUE THREAD

**Issue by philcryer** [mistio/mist-ce#1046]
I finally got around to installing mist-ce in docker-compose, and it looked really promising... until I saw the news about Dell. Please update the README.md with info of the Dell buyout, and that they're not going to support or release mist-ce going forward.  Also would be nice if you listed any active/promising forks going forward, I'm looking around but not seeing anything. It'd be a shame for all this work to just go away, again, I don't see a similar (open source) offering. Thanks for all the work, and here's hoping something positive comes of it.

**Comment by olljanat** [comment]
@philcryer I wonder if you found some good alternative for mist-ce?

There seems to be quite many tools but most of them are not updated anymore or free versions are very limited.

**Comment by philcryer** [comment]
> [@philcryer](https://github.com/philcryer) I wonder if you found some good alternative for mist-ce?
> 
> There seems to be quite many tools but most of them are not updated anymore or free versions are very limited.

Sorry to say I have not found an alternative that does what Mist-CE promised. It's a shame, it looks like something that was really polished. FWIW I did spin up the current docker-compose.yml but once registered couldn't get it to email me my account login so it feels dead in the water. I currently use Proxmox to organize and run my homelab but again, it doesn't have the featureset that Mist had. I'll keep looking though

**Comment by olljanat** [comment]
Yes, Mist-CE looks very promising for me too which quick look what I did on version [4.7.1](https://github.com/mistio/mist-ce/releases/tag/v4.7.1).

I also see that there is alpha release tag [5.0.0-alpha.14](https://github.com/mistio/mist-ce/releases/tag/v5.0.0-alpha.14) but no description what have been changed on it. Readme looks to be containing info about how to start development environment but at least api container looks to be failing on it.

Anyway, I ended up now creating GitHub organization [mistcommunity](https://github.com/mistcommunity) and forking all mistio repositories to there.

[mistcommunity/mist-ce](https://github.com/mistcommunity/mist-ce) has also Issues and Discussions features enabled in case people want to report known bugs or make feature requests.

Will not give promises that it will be finalized but will anyway try to get 3rd party modules updated and Mist-CE running after that.


Ping @therodfather I see that you have only [mist-ce repo for fork](https://github.com/therodfather/mist-ce) which is at least a bit ahead of upstream.

**Comment by olljanat** [comment]
> I wonder if you found some good alternative for mist-ce?

FYI, it looks to be Apache Libcloud which Mist uses. Other tools which uses it are also listed in https://libcloud.apache.org/whois-using.html

> Will not give promises that it will be finalized but will anyway try to get 3rd party modules updated and Mist-CE running after that.

Progressing slowly but looks still promising. Tracking issue available in https://github.com/mistcommunity/mist-ce/issues/1

**Comment by d-mo** [comment]
@philcryer Thank you for the suggestion and sorry for the delay in getting back to you. I didn't have the time to look into the Mist CE issues until now. I finally updated the README with information about the Dell acquisition. I also published a v4.7.2 release with security fixes contributed by @stolichnayer and @efchatz.

@olljanat Very happy to read about the mist-community fork. Please ensure you have the above fixes.

Closing this issue. Please let me know if I missed anything.

**Comment by olljanat** [comment]
@d-mo Welcome back and congratulations. You and your colleagues succeed build and sell Mist.io Inc.
Now if we manage to turn Mist CE to community maintained, pure open source project it would be perfect story.


As it seems that you are still/again in position to be able to update code here, I would like to ask that (and understand if you cannot answer to all of them):
* Are you still able to update mist.io site?
* Can we still keep using name Mist/Mist.io in open source version or is that now Dell's trademark?
* Are you able update [mist-ee](https://github.com/mistio/mist-ee) license also to Apache-2.0 and open those repositories related to SSO, RBAC and Insights to community?


When it comes to architecture:
* What are expected differences between 4.x and 5.x (latest code in master branch)? I see that at least:
  * HashiCorp Vault is used to to store secrets.
  * Python + some packages have been updated.
* Why CI was build with GitLab instead of GitHub workflows?
* I see that Libcloud was forked longtime ago heavily modified.
  * What are expected changes which hasn't been upstreamed?
  * I managed to get upstream version some level working https://github.com/mistcommunity/mist.api/commit/0997c3a745f85d9975757323a9cd88d19a15bf09 but most likely something is broken now.


> Very happy to read about the mist-community fork.

Community fork is just workaround as we assumed that no one has access to upstream anymore but of course preferably those changes should be provided back to here and released as 5.0 version if possible.

**Comment by d-mo** [comment]
Thank you @olljanat, I agree it would be excellent to see Mist continue as a community project.

Regarding naming and licensing: While there's no trademark on "Mist" or "Mist.io" as far as I know, Dell does own all relevant intellectual assets, including the logo. For the same reason, I'm unable to change the license of the RBAC, SSO, and Insights plugins. The safest approach for a community project would be adopting a new name—I've reserved cloudmeld.net and cloudmeld.io for this purpose, though I'm not certain of my future commitment to such a project at this point.

Architecture considerations for v5.x:

- Insights deprecation: This component was already being deprecated in v5.x, replaced by a new metering system leveraging VictoriaMetrics.
- RBAC & SSO recreation: The proprietary components can be rebuilt from scratch. I'm confident this could be accomplished in a weekend or two with some AI assistance.
- Vault integration: v5.x uses HashiCorp Vault for storing all secrets, and this implementation was nearly complete. However, a migration to OpenBAO should be considered for a fully open-source stack.
- Libcloud fork: We forked libcloud because we were rapidly fixing bugs, adding features, and modifying behavior—often for specific customer requirements with limited broader applicability. We contributed back upstream whenever possible, so your approach of using the latest upstream version makes the most sense.
- UI and API completion: The main remaining challenge for v5.x was completing the new UI implementation and v2 API while maintaining all existing functionality. This constraint could now be relaxed in a community fork.

Regarding CI: When we started Mist, GitHub Actions didn't exist. We began with Jenkins, then moved to GitLab CI, which enabled us to build an extensive automation and QA suite. However, ensuring we didn't break support for providers or minor features with each release remained our biggest development velocity bottleneck.

Moving forward: If the project gets revived, it should probably focus on a subset of the currently supported providers and features to maintain quality while enabling faster development cycles.
